irods/README.md

```diff
@@ -27,10 +27,10 @@ $ ./use-local-irods.sh
   $ cd irods
   $ ./use-local-irods.sh --persist
   ```
-  The `--persist` flag will create two new directories as `/home/${USER}/icat1` and `/home/${USER}/icat2` where iRODS vault and database files will be persisted. These directories will remain in place until the user manually destroys them.
+  The `--persist` flag will create a new directory as `/home/${USER}/icat1` where iRODS vault and database files will be persisted. These directories will remain in place until the user manually destroys them.
 
-The script will run, deploy and federate two iCAT servers, and modify the three aforementioned files to be configured to use the 
-newly created iRODS Docker servers. Once the script has completed, return back the the main `hydroshare` directory and run
+The script will run, deploy and federate an iCAT server, and modify the three aforementioned files to be configured to use the 

```

Comment:
how will it federate an iCAT server when there is only one server?

Reply:
good call, it wont! :)